NashTeamAlpha/BangazonWeb
# Added dynamic routing to the link in the navbar

## Pull Request Description

Modified the BaseView Model and the Menu Partial to change the routing based on whether or not there is an active customer. 

## Steps to Test

Include the applicable information from the list below here

1. git fetch
1. checkout `ddw-routing`
1. Make sure that when a customer has not been selected the route goes to Customers/New and pulls up the form
1. Make that when a customer has been selected the route goes to Customers/ShoppingCart

## Link to Feature Ticket

 #130
